ellama.el

```diff
@@ -181,6 +181,12 @@
   ;; Trim left first as `string-trim' trims from the right and ends up deleting all the code.
   (string-trim-right (string-trim-left text ellama--code-prefix) ellama--code-suffix))
 
+(defun ellama--generate-selected-provider-indicator ()
+  "Generate a string indicating the selected provider."
+  (if ellama-provider
+      (format "<!-- %s -->\n" (llm-ollama-chat-model ellama-provider))
```

Comment:
this will work for ollama only. Some users used it with other llm providers. We can't merge it like this.